apps/site/assets/ts/hooks/__tests__/useFilteredListTest.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import useFilteredList from "../useFilteredList";
+import { mount, ReactWrapper } from "enzyme";
+import stopData from "../../schedule/__tests__/lineDiagramData/simple.json";
```

Comment:
This fixture data is a bit of a [mystery guest](http://xunitpatterns.com/Obscure%20Test.html#Mystery%20Guest); I think this test would be more maintainable and readable if we instead used a short list of "example objects" tailored to the needs of the test, defined here in the test file. For example:

```js
const objectList = [
  { id: "a", stop: { name: "Origin" } },
  { id: "b", stop: { name: "Stop" } },
  { id: "c", stop: { name: "Destination" } }
];
```

We could also then include edge cases if we'd like to test them, without messing with the data for the line diagram tests; for example an item whose `stop` is null or undefined.